pkgs/games/factorio/default.nix

```diff
@@ -53,11 +53,11 @@ let
     x86_64-linux = let bdist = bdistForArch { inUrl = "linux64"; inTar = "x64"; }; in {
       alpha = {
         stable        = bdist { sha256 = "0b4hbpdcrh5hgip9q5dkmw22p66lcdhnr0kmb0w5dw6yi7fnxxh0"; version = "0.16.51"; withAuth = true; };
-        experimental  = bdist { sha256 = "1q66chnxsdlaz1bj3al62iikyxvknj1vkwh5bcc46favy4wpqpzz"; version = "0.17.52"; withAuth = true; };
+        experimental  = bdist { sha256 = "1h9mysxvsxjly9qc00k71asn9rv6qw9p0sxbz7wps4pj2b57jd21"; version = "0.17.66"; withAuth = true; };
```

Comment:
I got a diff sha:
```suggestion
        experimental  = bdist { sha256 = "057mb3ab85k0qxg158xmbhkygaz0xs3pqbs83h542xpv0629hc7y"; version = "0.17.66"; withAuth = true; };
```

Reply:
I just downloaded the package again and got the hash I originally put in. I also built using the script after deleting the downloaded file and it worked with my hash. You may have gotten a corrupted downloaded or not added a correct token

Reply:
unlikely:
```
[nix-shell:/home/jon/.cache/nix-review/pr-67766-1]$ nix build --keep-going -f ./nixpkgs/ factorio factorio-experimental
hash mismatch in fixed-output derivation '/nix/store/2cb10y9zgafyxv4rw3yphdi68cia8pgx-factorio_alpha_x64-0.17.66.tar.xz':
  wanted: sha256:1h9mysxvsxjly9qc00k71asn9rv6qw9p0sxbz7wps4pj2b57jd21
  got:    sha256:1qszpicbp03mc2ma4szpdb04nc1hzz3ldjyvsnqy1c6919ybib2d
cannot build derivation '/nix/store/y04gngfn7r2pfs19fmkzxkin0zqfxmww-factorio-alpha-0.17.66.drv': 1 dependencies couldn't be built
hash mismatch in fixed-output derivation '/nix/store/g77yqq1v108rgkclv2l1d6mj8hq5b0x4-factorio_alpha_x64-0.16.51.tar.xz':
  wanted: sha256:0b4hbpdcrh5hgip9q5dkmw22p66lcdhnr0kmb0w5dw6yi7fnxxh0
  got:    sha256:1z9ggmxk4bk87s0fbb13y3w4dbkh7ijrffvcb21izw5r5n68p9q6
cannot build derivation '/nix/store/albcywhndgzxh0955l7rm07jbn3nd0ir-factorio-alpha-0.16.51.drv': 1 dependencies couldn't be built
[0 built (2 failed)]
error: build of '/nix/store/albcywhndgzxh0955l7rm07jbn3nd0ir-factorio-alpha-0.16.51.drv', '/nix/store/y04gngfn7r2pfs19fmkzxkin0zqfxmww-factorio-alpha-0.17.66.drv' failed
```

Reply:
if nixpkgs finds a cache hit with the sha, it won't do any more verification on the contents. You most likely have a stale archive.

Reply:
I garbage collected my system and specifically used nix-store to delete it. Nix redownloaded it from the server. I also tried downloading directly and use nix-store --add to add it an get the hash